### PR TITLE
feat(fcm): enable direct_boot_ok as bool in Android config

### PR DESCRIPTION
**Pull request by akankshaJaiswal8**
### Discussion

  * https://github.com/firebase/firebase-admin-go/issues/634

### Testing

  go test -test.short firebase.google.com/go/...         
?       firebase.google.com/go/v4/errorutils    [no test files]
?       firebase.google.com/go/v4/integration/internal  [no test files]
?       firebase.google.com/go/v4/snippets      [no test files]
ok      firebase.google.com/go/v4       3.946s
ok      firebase.google.com/go/v4/appcheck      1.608s
ok      firebase.google.com/go/v4/auth  4.664s
ok      firebase.google.com/go/v4/auth/hash     3.035s
ok      firebase.google.com/go/v4/db    2.847s
ok      firebase.google.com/go/v4/iid   2.115s
ok      firebase.google.com/go/v4/integration/auth      4.185s
ok      firebase.google.com/go/v4/integration/db        4.624s
ok      firebase.google.com/go/v4/integration/firestore 5.664s
ok      firebase.google.com/go/v4/integration/iid       5.171s
ok      firebase.google.com/go/v4/integration/messaging 6.141s
ok      firebase.google.com/go/v4/integration/storage   6.411s
ok      firebase.google.com/go/v4/internal      7.386s
ok      firebase.google.com/go/v4/messaging     6.563s
ok      firebase.google.com/go/v4/storage       7.937s

### API Changes

  * This is not an API change but a change in existing AndroidConfig as suggested in the firebase official doc: https://firebase.google.com/docs/cloud-messaging/send-message?_gl=1*1o54gcz*_up*MQ..*_ga*NTk2NDY5NTMuMTcyMDY0NjAyMg..*_ga_CW55HF8NVT*MTcyMDY0NjAyMi4xLjAuMTcyMDY0NjAyMi4wLjAuMA..#send_using_the_fcm_v1_http_api

### CODE REVIEW

File: .idea/.gitignore
Comment (lahirumaramba) [inline .idea/.gitignore:1]: Let's remove the `.idea` directory from this PR

File: .idea/firebase-admin-go.iml
Comment (lahirumaramba) [inline .idea/firebase-admin-go.iml:1]: Let's remove the `.idea` directory and all the files under it from this PR

**Review by lahirumaramba — CHANGES_REQUESTED** [review]
Thank you for your contribution! Looks pretty good!
Let's remove the changes in `.idea` from this PR

File: .idea/.gitignore
Comment (akankshaJaiswal8) [reply .idea/.gitignore:1]: done

File: .idea/firebase-admin-go.iml
Comment (akankshaJaiswal8) [reply .idea/firebase-admin-go.iml:1]: done

**Review by lahirumaramba — APPROVED** [review]
Thank you for your contribution! LGTM!
Let me get the internal API approval process started before we can merge this change

**Pull request merged**
